Brain/Brain.ino

```diff
@@ -84,16 +84,19 @@ void setup() {
 }
 
 void loop() {
-  if (Serial.available() > 0) {
+  Serial.println("in loop");
+  if (Serial.available() >= 0) {
     int i = 0;
     while (Serial.available()) {
       command[i] = Serial.read();
       Serial.print(command[i]);
       i++;
     }
     Serial.println();
-    Serial.println(String(command));
-    int action = decode_received_packet(String(command), &valve);
+    //Serial.println(String(command));
+    //int action = decode_received_packet(String(command), &valve);
+    Serial.println("{20,12|12345}");
+    int action = decode_received_packet("{20,12|12345}", &valve);
```

Comment:
Can you revert these?
And the checksum should be 4 digits. Can you verify this works w/ a 4 digit checksum? try something hex too

Reply:
Reverted back and also tested that it does work with 4 digit hex numbers